middlewares/index.mjs

```diff
@@ -0,0 +1,12 @@
+import cookieParser from 'cookie-parser';
+import bodyParser from 'body-parser';
+
+const middlewares = [
+  bodyParser.urlencoded({ extended: true }),
+  bodyParser.json(),

```

Comment:
These bodyParser's methods are included with _express_ now
https://expressjs.com/en/guide/using-middleware.html#middleware.built-in

Reply:
It depends on express version

Reply:
Right. It's available with Express 4.16.0+. You are using 4.16.3